modules/RaptMediaNg/RaptMediaNg.json

```diff
@@ -0,0 +1,34 @@
+{
+	"mw.RaptMediaNg": {
+		"scripts": [
+			"resources/engine.min.js",
+			"resources/mw.RaptMediaNg.js"
+		],
+		"styles": "resources/RaptMediaNgPlugin.css",
+		"dependencies": [
+			"mw.KBaseComponent"
+		],
+		"kalturaPluginName": "raptMediaNg"
+	},
+	"mw.RaptMediaNgDurationLabel": {
```

Comment:
separating the stitched to playlist - do we need that label and alternative scrubber?

Reply:
I'm going to concur here. Other than the "core" `mw.RaptMediaNg` plugin all the others are simply shims that were required to patch bits of the UI to hide the fact that it was one large stitched playlist. As such is should be possible to completely remove them without any adverse effect.

I do see at least an argument for keeping them to preserve a closer to 1-to-1 mapping and minimize confusion when switching between the two different versions.